MediaAdminBundle/Transformer/MediaTransformer.php

```diff
@@ -84,9 +84,13 @@ public function getName()
      */
     protected function generateMediaUrl($key)
     {
-        $route = $this->generateRoute('open_orchestra_media_get',
-            array('key' => $key),
-            UrlGeneratorInterface::ABSOLUTE_PATH);
-        return $this->mediaDomain . $route;
+        if($key == null) {
+            return '/bundles/openorchestramediaadmin/images/no_image_available.jpg';
```

Comment:
You should use a constant for this string.

And maybe an application setting would be great :)